packages/node/src/views/composables.ts

```diff
@@ -1,4 +1,5 @@
 import { computed, onMounted, reactive, ref, shallowRef, watchEffect } from 'vue'
+import { useMagicKeys } from '@vueuse/core'
```

Comment:
Maybe we shouldn't use `VueUse`
https://github.com/webfansplz/vite-plugin-vue-devtools/pull/12#discussion_r1188654315

Reply:
Oh, Thanks ！ i`m remove && fixed